webpack/assets/javascripts/react_app/components/bookmarks/bookmarks.test.js

```diff
@@ -91,7 +91,7 @@ describe('bookmarks loading', () => {
 
 describe('bookmarks', () => {
   it('empty state', () => {
-    expect(toJson(render(setup().component))).toMatchSnapshot();
+    expect(toJson(mount(setup().component).find('Dropdown'))).toMatchSnapshot();
```

Comment:
using `render` triggers a warning: 
```
react-dom-server.browser.development.js:104 Warning: useLayoutEffect does nothing on the server, because its effect cannot be encoded into the server renderer's output format. This will lead to a mismatch between the initial, non-hydrated UI and the intended UI.
```
which cause by new implementation of `connect`:
https://github.com/reduxjs/react-redux/issues/1373

Reply:
We're not using server rendering, are we?